modules/oadp-using-enable-api-group-versions.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/advanced-topics.adoc
+
+
+:_content-type: PROCEDURE
+[id="oadp-using-enable-api-group-versions_{context}"]
+= Working with different Kubernetes API versions in the same OADP operation
+
+By default, Velero only backs up resources that use the `preferredAPI` Kubernetes version. However, Velero also includes a feature, link:https://velero.io/docs/v1.9/enable-api-group-versions-feature/[Enable API Group Versions], that overcomes this limitation. When enabled on the source cluster, this feature causes Velero to back up _all_ Kubernetes API group versions that are supported on the cluster, not only the preferred one.
+
+[NOTE]
+====
+As of August 2022, Enable API Group Versions is still in beta.
+====
+
+For example, consider the resource `horizontalpodautoscalers`, which falls under the `autoscaling` group and has 3 API versions - `v1`,`v2beta1`, and `v2beta2` - registered on the cluster, with the preferred API being `v1`.
```

Comment:
So technically this is the _plural_ for that. The actual API is called `HorizontalPodAutoscaler`. See the API docs: https://docs.openshift.com/container-platform/4.10/rest_api/autoscale_apis/horizontalpodautoscaler-autoscaling-v1.html#horizontalpodautoscaler-autoscaling-v1

The API group here is `autoscaling/v1`.

The plural is actually related to the REST API URL. I've always found that confusing, but it is this way _because REST_.